db/migrate_cards/20160413193247_source_type_set_cards.rb

```diff
@@ -0,0 +1,18 @@
+# -*- encoding : utf-8 -*-
+
+class SourceTypeSetCards < Card::Migration
+  def up
+    %w(File Link Text).each do |name|
+      create_source_type_set name
+    end
+  end
+
+  def create_source_type_set name
+    create_card!(
+      name: "#{name}+*source type",
+      type_id: Card::SetID,
+      content: '{"type":"source",' +
+               %("right_plus":["*source type", {"refer_to":"#{name}"}]})
+    )
+  end
+end
```

Comment:
this still confuses me.

It seems like:
1. there are Source cards with +*source type cards that determine their type
2. this creates a rule where ALL "+*source type" cards are sets.

Doesn't #2 override #1 and break things?